module/platform/OpenCR/HardwareIO/src/HardwareIO.cpp

```diff
@@ -284,7 +280,7 @@ namespace module::platform::OpenCR {
                         // Start the packet watchdog since the main loop is now starting
                         model_watchdog.disable();
                         model_watchdog.unbind();
-                        log<NUClear::WARN>("Packet watchdog enabled");
+                        log<NUClear::INFO>("Packet watchdog enabled");
                         packet_watchdog.enable();
```

Comment:
Here is where I would think the packet watchdog could be created, after the model watchdog is unbound